bin/merge_sces.R

```diff
@@ -132,6 +132,18 @@ read_trim_sce <- function(sce_file) {
 # get list of sces
 sce_list <- purrr::map(input_sce_files, read_trim_sce)
 
+# filter out libraries with fewer than 3 cells (causes errors with PCA)
+n_cells <- sce_list |> purrr::map_int(ncol)
+included_libs <- names(sce_list)[which(n_cells >= 3)]
+if (length(included_libs) < length(sce_list)) {
+  message(
+    "The following libraries have fewer than 3 cells and will be excluded from the merged object: ",
+    paste(setdiff(names(sce_list), included_libs), collapse = ", ")
+  )
+}
```

Comment:
might do this comparison first and use it for check? up to you

```suggestion
lib_diff <- setdiff(names(sce_list), included_libs)
if (length(lib_diff) > 0) {
  message(
    "The following libraries have fewer than 3 cells and will be excluded from the merged object: ",
    paste(lib_diff, collapse = ", ")
  )
}
```